_commands/caen/Caen-Connect-Tutorial.md

```diff
@@ -0,0 +1,149 @@
+
+## Make It Happen
+
+Now, to make this happen, all you need is a [sublime text](https://www.sublimetext.com) editor (More text editor options are to come as soon as I figure them out).
+
+
+#### Some Preparations
+1. Go ahead to [www.sublimetext.com](https://www.sublimetext.com) to get the latest version of sublime text installed on your machine (Mac/PC/Linux). Note that it doesn't matter whether you install version 2 or 3. They all works. (Skip if you already have done it in the past.)
+2. Install **Package Control**. [Here](https://packagecontrol.io/installation)'s how you could do it, or you could google it by yourself. (Skip if you already have done it in the past.)
+3. Hit `Cmd-Shift-P` if you are working on a Mac (`Ctrl-Shift-P` on PC) to open up the prompt. Type `Install Package` and hit `Enter`. In the next prompt, type **`SFTP`** and hit `Enter`. The installation will perform automatically. Restart your editor after this.
+
+#### Real Magic
+1. **Configurations**
+
+	Open up your **root/parent project directory** (where you put all your project folders for this course) and create a new json file named **`sftp-config.json`**. Open it up, and copy-paste in the following code:
+
+	```json
+	{    
+	    "type": "sftp",
+
+	    "save_before_upload": true,
+	    "upload_on_save": true,
+	    "sync_down_on_open": false,
+	    "sync_skip_deletes": false,
+	    "sync_same_age": true,
+	    "confirm_downloads": false,
+	    "confirm_sync": true,
+	    "confirm_overwrite_newer": false,
+
+	    "host": "login.engin.umich.edu",
+	    "user": "your-uniqname",
+	    "password": "your-password",
+
+	    "remote_path": "/afs/umich.edu/user/l/x/your-uniqname/path/to/your/project/folder",
+	    "ignore_regexes": [
+	        "\\.sublime-(project|workspace)", "sftp-config(-alt\\d?)?\\.json",
+	        "sftp-config(-alt\\d?)?\\.example.json",
+	        "sftp-settings\\.json", "/venv/", "\\.svn/", "\\.hg/", "\\.git/",
+	        "\\.bzr", "_darcs", "CVS", "\\.DS_Store", "Thumbs\\.db", "desktop\\.ini"
+	    ],
+
+	    "connect_timeout": 30,
+	}
+	```
+	Then, change all instances of `your-uniqname`s to your actual **uniqname**, change all instances of `your-password` to your **CAEN password**, change all instances of `path/to/your/project/folder` to where you want to put your project on CAEN. Now, go ahead and save it.
+
+	A sample config file can also be found here: [sftp-config.example.json](https://raw.githubusercontent.com/lxieyang/caen-connect-tutorial/master/sftp-config.example.json)
+
+	To explain a bit:
+	> `upload_on_save` will automatically upload the entire root/parent project folder to CAEN.
+
+	> `user` specifies whose CAEN account to upload your project to
+
+	> `password` specifies your password to CAEN
+
+	> `remote_path` specifies where your project folder is located on CAEN
+
+	> More configurations could be found here: [Sublime SFTP Settings](https://wbond.net/sublime_packages/sftp/settings#Settings)
+
```

Comment:
I'd prefer that these were normal bullet points. Blockquoting `>` is normally fine, but for some reason it renders a little odd on the website. The text here is bigger than the paragraph text.
![image](https://cloud.githubusercontent.com/assets/7095187/22857551/38d1a390-f075-11e6-9a96-5af40f46bbeb.png)